src/Aquifer.API/Endpoints/Resources/Content/SendForEditorReview/Endpoint.cs

```diff
@@ -0,0 +1,65 @@
+using Aquifer.API.Common;
+using Aquifer.API.Services;
+using Aquifer.Data;
+using Aquifer.Data.Entities;
+using FastEndpoints;
+
+namespace Aquifer.API.Endpoints.Resources.Content.SendForEditorReview;
+
+public class Endpoint(AquiferDbContext dbContext, IUserService userService, IResourceHistoryService historyService) : Endpoint<Request>
+{
+    public override void Configure()
+    {
+        Post(
+            //TODO: Delete assign-editor after next deploy
+            "/resources/content/{ContentId}/assign-editor",
+            "/resources/content/assign-editor",
+            "/resources/content/send-for-editor-review",
+            "/resources/content/{ContentId}/send-for-editor-review");
+        Permissions(PermissionName.AssignContent, PermissionName.AssignOverride, PermissionName.AssignOutsideCompany);
+    }
+    public override async Task HandleAsync(Request request, CancellationToken ct)
+    {
+        var user = await userService.GetUserFromJwtAsync(ct);
+        var permissions = ResourceStatusHelpers.GetAssignmentPermissions(userService);
+
+        await ResourceStatusHelpers.ValidateReviewerAndAssignedUser<Request>(request.AssignedUserId, request.AssignedReviewerUserId,
+            dbContext, user, permissions, ct);
+
+        var contentIds = request.ContentId is not null ? [(int)request.ContentId] : request.ContentIds!;
+
+        var draftVersions = await ResourceStatusHelpers.GetDraftVersions<Request>(contentIds, dbContext, ct);
```

Comment:
Again, validate statuses here. In fact, I think you could have this helper method take a list of statuses and only grab those statuses, and then the existing `ThrowError` can expand its error message to include not found in draft or correct status.

Reply:
y, we can follow the same thing we do in SendForCompanyReview

Reply:
just put it into a method with a parameter